### DISCUSSION TITLE
Updated Google event failed to save.....

### DISCUSSION THREAD

**Discussion by Hinakarah** [phw198/OutlookGoogleCalendarSync#1911] [General]
Hi @phw198,
Thank you for this tool. I really appreciate it. 
Generally it's been a smooth experience. 
I have a question about a popup I receive frequently "Updated Google event failed to save. Continue with synchronisation?"
I imagine it's the same event(s) that continue to trigger this. 
a) is there a way to disable the popup. I.e., is there a setting that lets synchronisation to continue without triggering the popup?
b) is there a way to update (or delete) the particular event (to stop receiving the popup)?

Thank you for your help with this. 

Unrelated, when updating events I have noticed that OGCS adds something to email addresses which then causes failures when calendar is updated as the email addresses have been appended with ogcs. Is there a way to deal with that? 

Thank you very much for creating this great little tool! 🙏😎👍🏆

**Comment by phw198** [comment]
The root cause of the popup needs identifying and fixing - are you able to provide your `OGcalsync.log` file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)) after you get it?

> Unrelated, when updating events I have noticed that OGCS adds something to email addresses which then causes failures when calendar is updated as the email addresses have been appended with ogcs. Is there a way to deal with that?

Yes, that's [carefully crafted behaviour](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs#why-am-i-getting-undeliverable-notifications-from-google-for-recipients-ogcs). If you want Google to spam attendees, then uncheck the "Cloak" option in settings.

**Comment by Hinakarah** [comment]
Good morning Paul,
Thank you very much for your response. I really appreciate it.


  1.  I attached the latest log file to this email.
  2.  By spamming, are you saying that they would get an email EACH time a sync happens, instead of ONLY when changes are made? Thank you for clarifying.
     *   When updating an event in Google, would a correct workaround be to remove the emails which contain the appended .ogs and add the original emails again?

Thank you again for your help with this Paul, I really appreciate it.

All the best,
Dieter

**Reply by phw198** [reply]
:paperclips: Unfortunately GitHub doesn't support submitting files and screenshots to the ticket when replying by email - please log in to GitHub to upload files.

No, not on every sync, but every time the Google event is modified it will email attendees. Not sure what you mean by a "correct workaround" - if you want attendees to be emailed, uncheck the cloaking option and perform a full sync; if you don't want them emailed, then OGCS is already doing that.

**Comment by LeRo93** [comment]
I have a similar problem.
After all, it is unthinkable to perform operations on normally created google events when, for example, a hotel or plane tickets are purchased. This should work normally. Thanks for the correction.
<img width="344" alt="in5KRw52Q3" src="https://github.com/user-attachments/assets/b7d29d96-cb9f-4963-bdb0-dc8fa5916cc4">

**Reply by wendellbeuke** [reply]
I have exactly the same problem. Updated event failed to save....

**Reply by phw198** [reply]
Potentially the same as #1919, which also means you haven't updated to a recent alpha.

**Comment by Hinakarah** [comment]
Good morning Paul,
I can confirm that 5 out of 6 “Bad request” issues are related to 3rd party apps creating calendar events and the visibility is “only me”. Once I go in and changed the visibility to “default visibility”, no more errors in syncing the events occurs.

The 6th sync issue may have to do with a series that got modified but I am not sure. I deleted the one event in the series which caused the issue – and now everything works as it should.

Thank you for your help. I really appreciate it.

All the best,
Dieter

**Reply by phw198** [reply]
> the visibility is “only me”

This is potentially extremely useful information! Thanks for sharing 👍🏻

**Comment by Hinakarah** [comment]
Hi Paul,
I do want the recipients to get notified, so I’ll change the cloak option.

Thank you for your support. I appreciate it.

All the best,
Dieter

**Comment by timelymooing** [comment]
Hi, just started using this after Caldavsynchronizer totally flipped out -- I really like the system, it's really simple and (mostly stable). I do run into the same issue with intermittent sync issues. The problem is that unless we're actively monitoring the system OGCS is operating on, we have no idea that we need to address issues until we notice our syncs have stopped. I'm using a remote computer to handle the calendar syncing essentially around the clock. 

In my case, sync issues have been intermittent: one time -- ran out of memory (which was kind of weird, the machine isn't super beefy but should have been enough to not run out of memory); the other time it was just I think an intermittent network problem. 

I know best case is to deal with the underlying cause, but it actually is the case that continuing in both of these cases would have allowed the problem to ultimately resolve itself. I wonder if this is something we could have on a timeout-- say 30 minutes (or something we can adjust) so that it would just go ahead and proceed after allowing any issues to stabilize (while also of course logging in the issue). The other is it would be great to get an email or other notification -- sent from me to me would work, but there's probably a fancier lemon prancy thing that can be done.

Thanks for the tool, I really like it! It's fantastic so far!

**Reply by phw198** [reply]
Thanks for the feedback and glad to have a convert from another application! 😉 

You're not wrong about the blocking error messages - it has been raised a long time ago (#1190) but other issues have always taken priority as it's not the simplest thing to resolve. Not keen on sending emails as this would require GMail scopes to be granted to use OGCS and that opens rather a can of worms, not least having to get Google to approve that usage by OGCS. Perhaps something like allowing users to configure [PushBullet](https://www.pushbullet.com/) notifications on popups could work... Or create a special calendar entry with a reminder set, so it pops up on user's phones... 🤔
